setup.py

```diff
@@ -1,12 +1,20 @@
 #!/usr/bin/env python
-# Copyright 2015 SKA South Africa (http://ska.ac.za/)
+# Copyright (c) 2015-2019, National Research Foundation (Square Kilometre Array)
 # BSD license - see COPYING for details
 
+from os import path
 from setuptools import setup, find_packages
 
+
+this_directory = path.abspath(path.dirname(__file__))
+with open(path.join(this_directory, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
 setup(
     name="katportalclient",
     description="A client for katportal.",
+    long_description=long_description,
+    long_description_content_type='text/markdown',
```

Comment:
That's nice - didn't know about that option.

Reply:
ye I picked it up when I ran `twine check dist/*` 
```bash
Checking distribution dist/katportalclient-0.1.0.tar.gz: warning: `long_description_content_type` missing.  defaulting to `text/x-rst`.
warning: `long_description` missing.
```
and also saw it here https://packaging.python.org/guides/making-a-pypi-friendly-readme/#including-your-readme-in-your-package-s-metadata